suprb/suprb.py

```diff
@@ -22,6 +23,18 @@
 
 
 class SupRB(BaseRegressor):
+    def __sklearn_tags__(self):
```

Comment:
what does it do and do we really want if before the documentation? Is this the sklearn standard guideline?